CEDS/MSSQL/0037-View-FactK12ProgramParticipationsDim-Create.sql

```diff
@@ -0,0 +1,157 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+IF EXISTS (
+        SELECT 1
+        FROM INFORMATION_SCHEMA.VIEWS
+        WHERE TABLE_SCHEMA = 'analytics' AND TABLE_NAME = 'ceds_FactK12ProgramParticipation'
+        )
+BEGIN
+    DROP VIEW analytics.ceds_FactK12ProgramParticipation;
+END;
+GO
+
+CREATE VIEW analytics.ceds_FactK12ProgramParticipation AS
+	WITH FactK12ProgramParticipation
+	AS (
+        SELECT
+        (
+            SELECT
+            CONCAT(EducationOrganizationId, '-', ProgramTypeDescriptorId,'-', ProgramEducationOrganizationId)
+            FROM edfi.StudentSpecialEducationProgramAssociation
+            GROUP BY EducationOrganizationId, ProgramTypeDescriptorId, ProgramEducationOrganizationId
+	    ) AS FactK12ProgramParticipationId,
+           SchoolYearKey
+           ,'' AS DateId
+           ,'' AS DataCollectionId
+           ,SeaIdentifierSea AS 'SeaId'
+           ,LeaIdentifierSea AS 'IeuId'
```

Comment:
Maybe what we have to return here is `IeuOrganizationIdentifierSea` instead. Can you double check?